libcxx/docs/Status/Cxx2cIssues.csv

```diff
@@ -113,7 +113,7 @@
 "","","","","",""
 "`LWG3578 <https://wg21.link/3578>`__","Iterator SCARYness in the context of associative container merging","2025-02 (Hagenberg)","","",""
 "`LWG3956 <https://wg21.link/3956>`__","``chrono::parse`` uses ``from_stream`` as a customization point","2025-02 (Hagenberg)","","",""
-"`LWG4172 <https://wg21.link/4172>`__","``unique_lock`` self-move-assignment is broken","2025-02 (Hagenberg)","","",""
+"`LWG4172 <https://wg21.link/4172>`__","``unique_lock`` self-move-assignment is broken","2025-02 (Hagenberg)","|Complete|","21",""
```

Comment:
No change requested. I found that [LWG4172](https://cplusplus.github.io/LWG/issue4172) reverted [LWG2104](https://cplusplus.github.io/LWG/issue2104) so the latter should be Nothing To Do. But we've already removed the corresponding issue pages in #105514.